src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -164,6 +171,137 @@ public Task GetTopology()
+                throw new BadRequestException(errorMessage);
+
+            ServerStore.EnsureNotPassive();
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            {
+                var json = await context.ReadForMemoryAsync(RequestBodyStream(), "read-conflict-resolver");
+                var conflictResolver = (ConflictSolver)EntityToBlittable.ConvertToEntity(typeof(ConflictSolver), "convert-conflict-resolver", json, DocumentConventions.Default);
+
+                using (context.OpenReadTransaction())
+                {
+                    var databaseRecord = ServerStore.Cluster.ReadDatabase(context, name, out _);
+
+                    var (index, _) = await ServerStore.ModifyConflictSolverAsync(name, conflictResolver);
+                    await ServerStore.Cluster.WaitForIndexNotification(index);
+
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.Created;
+
+                    using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                    {
+                        context.Write(writer, new DynamicJsonValue
+                        {
+                            ["RaftCommandIndex"] = index,
+                            ["Key"] = name,
+                            [nameof(DatabaseRecord.ConflictSolverConfig)] = databaseRecord.ConflictSolverConfig.ToJson()
+                        });
+                        writer.Flush();
+                    }
+                }
+            }
+        }
+
+        [RavenAction("/periodic-backup/status", "GET", AuthorizationStatus.ValidUser)]
+        public Task GetPeriodicBackupStatus()
+        {
+            var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
+
+            if (TryGetAllowedDbs(name, out var _, requireAdmin: false) == false)
+                return Task.CompletedTask;
+
+            var taskId = GetLongQueryString("taskId", required: true);
+            if (taskId == 0)
+                throw new ArgumentException("Task ID cannot be 0");
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            using (var statusBlittable =
+                ServerStore.Cluster.Read(context, PeriodicBackupStatus.GenerateItemName(name, taskId.Value)))
+            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+            {
+                writer.WriteStartObject();
+                writer.WritePropertyName(nameof(GetPeriodicBackupStatusOperationResult.Status));
+                writer.WriteObject(statusBlittable);
+                writer.WriteEndObject();
+                writer.Flush();
+            }
+
+            return Task.CompletedTask;
+        }
+
+        [RavenAction("/connection-strings", "GET", AuthorizationStatus.DatabaseAdmin)]
```

Comment:
I'm pretty sure that this kind of URL doesn't actually apply the right permissions